src/App.tsx

```diff
@@ -15,36 +15,109 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortField {
+  alphabet = 'alphabet',
+  length = 'length',
+}
+
 export const App: React.FC = () => {
+  const [goods, setGoods] = useState<string[]>(goodsFromServer);
+  const [isActive, setIsActive] = useState('');
```

Comment:
```suggestion
  const [isActive, setIsActive] = useState(SortField.default);
```
create the default field